modules/010-cluster-autoscaler-crd/crds/machinetypescatalogs.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: machinetypescatalogs.deckhouse.io
+  labels:
+    heritage: deckhouse
+    module: cloud-discovery-crd
+spec:
+  group: deckhouse.io
+  scope: Cluster
+  names:
+    plural: machinetypescatalogs
+    singular: machinetypescatalog
+    kind: MachineTypesCatalog
+  preserveUnknownFields: false
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      schema: &schema
+        openAPIV3Schema:
+          type: object
+          description: |
+            Contains information about discovered data from cloud.
+          required:
+            - instanceTypes
+          properties:
+            instanceTypes:
+              description: List of available instance types.
+              type: array
+              items:
+                type: object
+                properties:
+                  name:
+                    type: string
+                    description: Instance type name.
+                  memory:
+                    type: integer
+                    format: int64
```

Comment:
It is better to use the usual Kubernetes quantity instead of plain integers. 
See for an inspiration

https://github.com/deckhouse/deckhouse/blob/df078d560f8b41fdfedc3c06eab072758fbb355f/modules/010-vertical-pod-autoscaler-crd/crds/verticalpodautoscaler.yaml#L83-L92

Reply:
done